src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/identifiers/PNCIdentifier.kt

```diff
@@ -1,65 +1,76 @@
 package uk.gov.justice.digital.hmpps.personrecord.model.identifiers
 
+import org.apache.commons.lang3.builder.EqualsBuilder
+import org.apache.commons.lang3.builder.HashCodeBuilder
 import java.time.LocalDate
 
-interface PNCIdentifier {
+class PNCIdentifier(val pncId: String) {
+
+  val valid: Boolean
+    get() = pncId.isNotEmpty()
```

Comment:
do we even need this method? It doesn't look as if anything is calling it apart from the test

Reply:
Removed 👍